compiler/x/amd64/codegen/AMD64FastCallConfig.hpp

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
Are these files marked as copyright 2000 because they contain excerpts from other files that have copyright 2000?

Reply:
Thank you for point it out. In this case, it was a mistake; X86FastCallLinkage.hpp and X86FastCallLinkage.cpp contain modified code from OpenJ9, which have copyright 2000.

Reply:
Updated.